pkg/auth/keymanager.go

```diff
@@ -156,7 +154,7 @@ func (km *KeyManager) fetchKeys(keysEndpointURL string) ([]*PublicKey, error) {
 	bodyString := buf.String()
 	// if status code was not OK, bail out
 	if res.StatusCode != http.StatusOK {
-		km.logger.Println(map[string]interface{}{
+		log.Info(nil, "status not OK", map[string]interface{}{
```

Comment:
`log.Info()` doesn't work with maps. It works with slices.
So it should be something like
```
		log.Info(nil, "unable to obtain public keys from remote service: status not OK",
			"response_status", res.Status,
			"response_body", bodyString,
			"url", keysEndpointURL)
```

But see my other comment: https://github.com/codeready-toolchain/registration-service/pull/38/files?file-filters%5B%5D=.go&file-filters%5B%5D=.mod#diff-1135f265d25e96e557b8114567653176R58

Reply:
fixed here: 00bf17d961e783ddd6ad166872fdd5903f82a7c1